### ISSUE TITLE
[Bug]: Aoetec Z-Stick Gen5+ no configuration and does not support smart start

### ISSUE THREAD

**Issue by D3v01dZA** [zwave-js/zwavejs2mqtt#2375]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Zwavejs2Mqtt version

6.7.1

### ZwaveJS version

9.0.2

### Describe the bug

On a clean ZWaveJS2MQTT I am using an Aoetec Z-Stick Gen5+ (https://aeotec.com/z-wave-usb-stick/index.html).

While using it I attempted to use SmartStart, only to be greeted with an error that SmartStart is not suported (directly from inside of the UI, I checked in Home Assistant too and the option to scan is not there). I think went to look at the stick in the UI and noticed that it has no configuration options available on it. The UI also says "Z-Stick Gen5 USB Controller" with no mention of Plus or + which does confuse me, although as far as I know that it shares a product id with the non + version which is confusing me since the stick does say Gen5+ on the box and on itself...

[Log File](https://github.com/zwave-js/zwavejs2mqtt/files/8496497/log.log)

Dashboard
![Image](https://user-images.githubusercontent.com/10627822/163583992-eccd1bab-2202-44b7-8e64-5ca0fa60f6ba.PNG)

### To Reproduce

Use an Z-Stick Gen5+

### Expected behavior

- I expect to be able to configure the stick
- I expect SmartStart to work

**Comment by mundschenk-at** [comment]
You probably need to update the firmware of the stick. I've done so myself, but TIL that for some reason, Aeotec have withdrawn the firmware update support article. You will need to open a support ticket with them to get access to the update and installation instructions.

**Comment by robertsLando** [comment]
cc @AlCalzone . I don't remember from which version SS is supported in Gen5+ but this seems the last one so there should be something else wrong here

**Comment by D3v01dZA** [comment]
Looking at their documentation it's supposedly from release, one of the things the + added was specifically SS. I'm currently testing it because of 700 series woes (even after the latest firmware updates).

**Comment by mundschenk-at** [comment]
SmartStart should work fine with that firmware version, I have not had any problems there with my upgraded stick (for the same reason).

**Comment by D3v01dZA** [comment]
This is what I figured the case was, I'm currently wondering if I should stick to trying to figure it out or just return it to Amazon...

**Comment by cwich** [comment]
I have the exact same Aeotec Z-Stick Gen5+ (FW 1.2, SDK 6.7.0) running in the latest Zwavejs2Mqtt 6.7.1 / Z-Wave JS 9.0.2 docker version. Today I tried to add another Z-Wave device using Smart Start but it does not work any more.

Last Time I added a Z-Wave device About three weeks ago it was working with the same stick but an earlier zwavejs2mqtt version. I'm using Home Assistant and was able to scan the Smart Start QR-Code with my iPad and die Home Assistant app.

Could it be that the Smart Start support war broken a couple versions before?

**Comment by mundschenk-at** [comment]
> Last Time I added a Z-Wave device About three weeks ago it was working with the same stick but an earlier zwavejs2mqtt version. I'm using Home Assistant and was able to scan the Smart Start QR-Code with my iPad and die Home Assistant app.
> 
> Could it be that the Smart Start support war broken a couple versions before?

Interesting. It works fine for me, but I'm still on 6.6.2 (using the Home Assistant Community Add-on).

**Comment by kpine** [comment]
Anyone have a startup log for a version of z2m prior to 6.7.0? One that shows the controller firmware versions.

SmartStart requires SDK v6.81, and the driver checks for that minimum version. The Gen5+ is supposed to be v6.81.6. From the logs, the stick is reporting v6.7.0, but that response might be invalid. Prior driver versions queried the SDK version via a different method.

**Comment by D3v01dZA** [comment]
I've been messing with it for a while now, I could quite easily start up an older version but it will only be tomorrow.

@cwich if you look under the device at the configuration do you see any options (should be one for disabling the LED for example) or is it empty with only the custom configuration options?

**Comment by kpine** [comment]
USB controllers don't have config settings.

**Comment by D3v01dZA** [comment]
I'm really not an expert but I don't think that's right, see https://github.com/zwave-js/node-zwave-js/issues/1208#issuecomment-747718608

**Comment by kpine** [comment]
USB controllers that Z-Wave JS communicates with over the Serial API don't have config settings. Keep reading the comments you linked to, you stopped a bit short.

If you want to disable the LED, you have to use the Serial API. You can either send a [custom command](https://zwave-js.github.io/node-zwave-js/#/usage/custom?id=send-a-custom-message-to-the-controller) using z2m (Driver function in Advanced), or use the command line: `echo -e -n "\x01\x08\x00\xF2\x51\x01\x00\x05\x01\x51" > /dev/serial/by-id/usb-0658_0200-if00`.

**Comment by D3v01dZA** [comment]
I must admit I was confused by that thread so I'll defer to you here, but it definitely seems that the comment I linked has the options in the GUI which only adds to my confusion with what you're saying and what I see there (and the options in the comment are exactly the ones that Aoetec advertises). I don't really mind the config not being available, just thought it was prudent to include everything I thought was suspect when I opened the bug.

**Comment by kpine** [comment]
The screenshot in that issue is from **ZWave**2MQTT, not **ZWaveJS**2MQTTT. These are different software, the former based on OpenZWave. Z-Wave JS doesn't interview serial-api controlled controllers, so config settings are never populated. OZW always populates the config parameters without considering if it makes sense or not.

If you did have a Gen5 as a secondary controller, it would be interviewed like all the other nodes and the config settings populated and controllable (although Z-Wave JS doesn't officially support secondary controllers).

**Comment by D3v01dZA** [comment]
Gotcha, thanks for the clarification!

**Comment by cwich** [comment]
@D3v01dZA : the config ist empty besides the custom option
<img width="1354" alt="Bildschirmfoto 2022-04-17 um 09 50 01" src="https://user-images.githubusercontent.com/55660736/163705683-11382902-f7aa-4d77-8424-0aabf87fbbcb.png">
.

**Comment by prilly-dev** [comment]
> Anyone have a startup log for a version of z2m prior to 6.7.0? One that shows the controller firmware versions.
> 
> SmartStart requires SDK v6.81, and the driver checks for that minimum version. The Gen5+ is supposed to be v6.81.6. From the logs, the stick is reporting v6.7.0, but that response might be invalid. Prior driver versions queried the SDK version via a different method.

Yes thiks is a bug in zwave, it shows SDK 6.7.0 but in fact should it be 6.0.7 that is the same as sdk V6.81

i just did the upgrade of my gen5 from v1.01 to v1.02 firmware and went trough support at aeotec, indeed the stick has right firmware confirmed with zwave pc controller.

**Comment by AlCalzone** [comment]
If this is a driver bug, I'm still going to need the startup logs that were requested above. 🙏

**Comment by D3v01dZA** [comment]
@AlCalzone included startup logs from 6.6.2, where I also added a provisioning entry

[Log File](https://github.com/zwave-js/zwavejs2mqtt/files/8505884/log.log)

**Comment by AlCalzone** [comment]
@D3v01dZA I'm preeeetty sure that the driver recognized the stick to support Smart Start:
```
2022-04-18T13:21:33.108Z CNTRLR   Smart Start listening mode enabled
```

**Comment by kpine** [comment]
In 6.6.2 it did correctly, but not 6.7.0 (v9).

```
2022-04-15T14:22:56.024Z DRIVER « [RES] [GetProtocolVersion]
                                    payload: 0x000607000061
2022-04-15T14:22:56.025Z CNTRLR   received protocol version info:
                                    protocol type:             Z-Wave
                                    protocol version:          6.7.0
                                    appl. framework build no.: 97
2022-04-15T14:22:56.026Z CNTRLR   supported Z-Wave features: 
2022-04-15T14:22:56.026Z CNTRLR   querying controller capabilities...
```

The difference being the use of GetProtocolInfo in 6.7.0.

**Comment by AlCalzone** [comment]
Oh, sorry - must have overlooked that part.
